db/migrate/20170421162639_change_originality_report_score_null.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeOriginalityReportScoreNull < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :originality_reports, :originality_score, true
```

Comment:
It looks like schema.rb already lacks a not null constraint for originality_score